js_modules/dagit/packages/core/src/assets/AssetNodeList.tsx

```diff
@@ -53,6 +53,6 @@ const Container = styled(Box)`
 
 const AssetNodeWrapper = styled.div`
   cursor: pointer;
-  width: 240px;
+  width: 260px;
```

Comment:
This allows room for dates displaying the previous year here: 

<img width="311" alt="image" src="https://user-images.githubusercontent.com/1037212/213587053-73bab14a-56a9-4034-bd2b-c013e07790e0.png">

Reply:
I wonder if instead of the exact date we just show like "1 day ago" or something, and the on hover we show the full date? Might save us some space here

Reply:
Yeah I think that might be nice - now that we show the word "Materialized" we have had to make these bigger in a few places recently.